apollo/server/models/SocSetting.ts

```diff
@@ -55,6 +59,7 @@ export const SocSettingFactory = (sequelize: Sequelize): typeof SocSetting => {
         type: DataTypes.STRING,
         allowNull: false,
       },
+      updatedAt: DataTypes.DATE,
```

Comment:
does this cause any errors?

Reply:
Nope.
BTW this declaration is necessary to keep TypeScript happy

Reply:
actually, sequalize will do this automatically, so I think only line 14 is necessary.
Line 62 is not needed and line 20 should ignore "updatedAt"